scripts/changelog.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+tfile=$(mktemp /tmp/openapi-CHANGELOG-XXXXXX)
+github-changelog-generator -org danaelhe -repo openapi >"$tfile"
```

Comment:
no idea if your fork is intentional or not

```suggestion
github-changelog-generator -org digitalocean -repo openapi >"$tfile"
```